frontend/cypress/docker-compose.yml

```diff
@@ -63,6 +63,7 @@ services:
       REACT_APP_OKTA_URL: http://cypress:8088
     volumes:
       - ../:/frontend/
+      - ../../.git/:/frontend/.git
```

Comment:
Does this help with running the health check e2e test in the compose setup?

Reply:
Yes. This resolves an issue where the frontend is not setting REACT_APP_CURRENT_COMMIT when starting in a docker container and resolves an issue with a failing Cypress health check test.

It should have been included in my last cypress PR, but seems to have been pulled out.